src/pages/en/guides/deploy/cloudflare.md

```diff
@@ -28,7 +28,7 @@ To get started, you will need:
     - **Framework preset**: `Astro`
     - **Build command:** `npm run build`
     - **Build output directory:** `dist`
-    - **Environment variables (advanced)**: Currently, Cloudflare Pages supports `NODE_VERSION = 12.18.0` in the Pages build environment by default. Astro requires `14.15.0`, `v16.0.0`, or higher. You can add an environment variable with the **Variable name** of `NODE_VERSION` and a **Value** of a [Node version that’s compatible with Astro](/en/install/auto/#prerequisites) or by specifying the node version of your project in a `.nvmrc` or `.node-version` file.
+    - **Environment variables (advanced)**: By default, Cloudflare Pages uses `NODE_VERSION = 12.18.0`. However, Astro [requires](/en/install/auto/#prerequisites) `14.18.0`, `v16.12.0`, or higher. Use a `.nvmrc.` or `.node-version` file to manually specify a version, or add an environment variable with a **Variable name** of `NODE_VERSION` and a **Value** of `14.18.0`, `v16.12.0`, or higher.
```

Comment:
There are a lot of options here (nvmrc, node-version, env variable, v14, v16…). How do we feel about reducing that a little?

[Node 14 will reach end-of-life in April](https://nodejs.org/en/about/releases/), so we can probably safely recommend most folks choose 16 unless they really need 14. I also don’t see a reason to document both `.node-version` and `.nvmrc`.

I’ve also tried to simplify the text a little. Let me know what you think!

```suggestion
    - **Environment variables (advanced)**: By default, Cloudflare Pages uses Node.js 12.18.0, but Astro [requires a higher version](/en/install/auto/#prerequisites). Add an environment variable with a **Variable name** of `NODE_VERSION` and a **Value** of `v16.12.0` or higher to tell Cloudflare to use a compatible Node version. Alternatively, add a `.nvmrc` file to your project to specify a Node version.
```